src/App.vue

```diff
@@ -23,9 +23,13 @@
               <li v-bind:class="{ 'active': this.$router.currentRoute.path === '/mint/assets' }">
                 <router-link to="/">General Products</router-link>
               </li>
-              <li v-bind:class="{ 'active': this.$router.currentRoute.path === '/mint/real-estate' }">
-                <router-link to="/mint/real-estate">Real Estate</router-link>
+              <li v-bind:class="{ 'active': this.$router.currentRoute.path === '/mint/video-latino' }">
+                <router-link to="/mint/video-latino">Video Latino</router-link>
               </li>
+              <!-- Route un-used for now -->
+<!--              <li v-bind:class="{ 'active': this.$router.currentRoute.path === '/mint/real-estate' }">-->
+<!--                <router-link to="/mint/real-estate">Real Estate</router-link>-->
+<!--              </li>-->
```

Comment:
I'll find out if they still intend to use this - I think they will say maybe so we should keep it visible for now

Reply:
Ok can keep visible no probs